### ISSUE TITLE
limit of detection

### ISSUE THREAD

**Issue by timbrisc** [ucum-org/ucum#216]
*Issue migrated from trac ticket # 5818*

**component:** organization | **priority:** minor | **keywords:** LOD

#### 2022-06-07 20:22:30: william.hess@fda.hhs.gov created the issue

___
> UCUM Board needs to provide input on this.  UCUM already has "Limit of flocculation" so adding a UCUM term called "limit of detection" would be somewhat consistent.  Limit of detection, LOD is the lowest concentration that can be measured (detected) with statistical significance by means of a given analytical procedure.20 It is a measured quantity value, obtained by a given measurement procedure, for which the probability of falsely claiming the absence of a component in a material is β, given a probability α of falsely claiming its presence.1 Practically, it is determined on the basis of the noise level: it is most often three times as high.  Please see https://www.sciencedirect.com/topics/nursing-and-health-professions/limit-of-quantitation#:~:text=Limit%20of%20quantification%2C%20LOQ%20stands,accuracy%2C%20precision%2C%20and%20uncertainty.
> 
> This new concept is needed to support the U.S. Food and Drug Administration Pharmaceutical Quality Chemistry Manufacturing and Control (PQ/CMC) initiative. Please see https://www.regulations.gov/document/FDA-2022-N-0297-0001, which in part, states:
> 
> "PQ/CMC is a term used to describe manufacturing and testing data of pharmaceutical products. PQ/CMC encompasses topics such as drug stability, quality specification, batch formula, and batch analysis, which are important aspects of drug development. PQ/CMC plays an integral part in the regulatory review process and life cycle management of pharmaceutical products. The development of a structured format for PQ/CMC data will enable consistency in the content and format of PQ/CMC data submitted, thus providing a harmonized language for submission content, allowing reviewers to query the data, and, in general, contributing to a more efficient and effective regulatory decision-making process by creating a standardized data dictionary.
> 
> The impetus for this standardization effort was the provisions from the 2012 Food and Drug Administration Safety and Innovation Act (Pub. L. 112-144), which authorized the Agency to require certain submissions to be submitted in a specified electronic format. PQ/CMC standardization supports FDA's regulatory needs in receiving structured and standardized data in pharmaceutical quality and includes two objectives: (1) To standardize the pharmaceutical quality data that is currently received by FDA in eCTD Module 3 from the sponsoring organizations, and (2) to use these structured elements and develop a FHIR data exchange solution."
>

**Comment by timbrisc** [comment]
#### 2022-06-07 20:37:05: william.hess@fda.hhs.gov edited the issue description

**Comment by dr-shorthair** [comment]
This concept is not a **unit-of-measure**, it is a quality measure or **part of the definition of a measurement-procedure**.

**Comment by gschadow** [comment]
I agree with Simon (?) that this is probably not a unit as such. Limit of detection is device dependent, certainly it is "procedure" dependent, but not in that abstract sense in which all units are defined based on some procedure, but dependent on a particular procedure reduced into practice, considering all particular instances of instruments (e.g., if the measurement finally is reduced to an electrical potential, then it is the particular voltmeter and its reliability and error and impedance and all sorts of characteristics which determine that limit of detection.

So I can't see how that would be a unit of measure.

Except in a case where the limit of detection is itself averaged over a large number of subjects. For example, some psycho-physiologic measurement scales (there was something with S. S. Stevens) may be based on the limit of detection or limit of difference. I don't recall now all the specifics. But in those cases the scale is grounded in what the average biological sensor may discern and that is not changed by technology, therefore it could serve as a unit of measure.
